src/t8_forest/t8_forest_adapt.cxx

```diff
@@ -67,8 +67,12 @@ t8_forest_adapt_coarsen_recursive (t8_forest_t forest, t8_locidx_t ltreeid,
   element = t8_element_array_index_locidx (telements, *el_inserted - 1);
   /* TODO: This assumes that the number of children is the same for each
    *       element in that class. This may not be the case. */
```

Comment:
is this TODO comment still valid or adressed now?

Reply:
I adressed it now and deleted the comment.